src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindInFilesDialog.ui.xml

```diff
@@ -32,8 +32,10 @@
          <rw:LabeledTextBox textBoxId="{ElementIds.getFindFilesText}" ui:field="txtSearchPattern_" 
                             labelText="Find:" enableSpellcheck="false" styleName="{style.pattern}"/>
          <g:CheckBox ui:field="checkboxCaseSensitive_" text="Case sensitive"/>
-         &#x00a0;&#x00a0;&#x00a0;&#x00a0;
+         &#x00a0;
          <g:CheckBox ui:field="checkboxRegex_" text="Regular expression"/>
+         &#x00a0;
+         <g:CheckBox ui:field="checkboxIgnore_" text="Exclude ignore files"/>
```

Comment:
We should hide this checkbox since it doesn't do anything yet. You can just set `visible="false"` for now so you don't have to remove any of the code that references it.

Reply:
Fixed